screen/Running/components/Header.js

```diff
@@ -53,7 +53,9 @@ const Header = ({
           onFinish={onFinish}
         />
       )}
-      {mode === 'solo' && <ValueWithUnit value={speed} unit="km/h" />}
+      {mode === ('solo' || 'survival') && (
```

Comment:
mode가 'solo'이거나 'survival'일 때 true가 되는 조건문을 의도하신게 맞으실지요? 이 조건문은 mode가 'survival'일때 false를 반환하게 됩니다. 아래와 같이 수정해보시는건 어떠실까요?

```suggestion
      {(mode === 'solo' || mode === 'survival') && (
```

Reply:
감사합니다. 피드백 해주신 부분의 의도가 맞습니다. or 논리 연산자에서 실수를.. 피드백 감사합니다.